README.md

```diff
@@ -51,6 +51,8 @@ These key / values can be, but do not need to be included in the `info.json` fil
 
 - `bundle_id` - (optional) It should match to the bundle id set on the Apple Developer website for this app. If it's not set it will default to: "org.kiwix.custom.{brand_name}", where the brand name is the name of the folder, eg: "org.kiwix.custom.dwds".
 
+- `support_url` - (optional) The _"Support \\(app_name)"_ item in the sidebar by default links to: "https://kiwix.org/get-involved/". This link target can be changed by setting `support_url`. It has to be a vaild URL. If setting it to an invalid URL, eg. empty value: `"support_url": ""`, the _"Support \\(app_name)"_ item will be hidden in the app.
```

Comment:
Can we use `{brand_name}` instead of `\\(app_name)` as it is for `bundle_id` ?

Reply:
We literally take the value from the json property: "app_name" (see on line 31 above).